src/SignalR/common/Shared/AsyncEnumerableAdapters.cs

```diff
@@ -32,9 +32,11 @@ await foreach (var item in channel.ReadAllAsync(cancellationToken))
             }
         }
 #else
+        // System.Threading.Channels.ReadAllAsync() is not available on netstandard2.0 and netstandard2.1
+        // But this is the exact same code that it uses
```

Comment:
Was just going to ask for this after our convo 👍

Reply:
![](https://media.giphy.com/media/3o7TKy8m0Q4yhwYGAw/giphy.gif)